recipe/migrations/jpeg9e.yaml

```diff
@@ -0,0 +1,13 @@
+migrator_ts: 1676157958
+__migrator:
+  kind: version
+  migration_number: 1
+  bump_number: 1
+  commit_message: "Rebuild for jpegturbo migration"
+  # limit the number of prs to do the initial test
+  pr_limit: 1
+
+# This migration is matched with a minimigrator that will
+# replace jpeg with libjpegturbo
+jpeg:
+  - '9e'
```

Comment:
Are you sure `9e` will work? Have you tried locally?

Reply:
Nvm, let's use a non-existing version here. `10`?

Reply:
sure

Reply:
Renamed the migrator filename to match this change. Hope that is ok :)

Reply:
Yes thank you. I suggested a change https://github.com/regro/cf-scripts/pull/1604/files

Reply:
It should be something like `libjpeg_turbo.yaml`.
`jpeg9e.yaml` and `jpeg10.yaml` are both misleading.

Reply:
Renamed to `jpeg_to_libjpeg_turbo`